app/controllers/api/v0/classified_listings_controller.rb

```diff
@@ -44,35 +45,31 @@ def update
 
       def process_no_credit_left
         msg = "Not enough available credits"
-        render json: [{ error: msg }], status: :payment_required
+        render json: { error: msg, status: 402 }, status: :payment_required
```

Comment:
Cool :)

Reply:
I have done so for the other json calls too

Reply:
I was referring to the choice of using HTTP Payment Required as a return code :)